rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/NodeSourceConfigurationParser.java

```diff
@@ -42,6 +42,8 @@
 
 public class NodeSourceConfigurationParser {
 
+    private static final String JSON_FILE_ERROR_MESSAGE = "The imported file is not a valid JSON file";
+
     public JSONValue parseJSON(String jsonStr) {
         try {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Catch Exception instead of Throwable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)